__tests__/app.test.js

```diff
@@ -181,4 +181,69 @@ describe("PATCH /api/reviews/:review_id", () => {
         expect(msg).toBe("invalid type please check your input");
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/reviews", () => {
+  test("should return status 200 and an array of all reviews sorted by creat date desc", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((res) => {
+        const review = res.body.review;
+        expect(Array.isArray(review)).toBeTruthy();
+        expect(review).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        review.forEach((index) => {
+          expect(index).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              category: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_body: expect.any(String),
+              review_img_url: expect.any(String),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("should return status 200 and an array of all reviews in a specific category", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction")
+      .expect(200)
+      .then((res) => {
+        const review = res.body.review;
+        expect(Array.isArray(review)).toBeTruthy();
+        review.forEach((index) => {
+          expect(index).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              category: "social deduction",
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_body: expect.any(String),
+              review_img_url: expect.any(String),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("should return status 404 if category was not found", () => {
+    return request(app)
+      .get("/api/reviews?category=lala")
+      .expect(404)
+      .then((res) => {
+        const msg = res.text;
+        expect(msg).toBe("Sorry can't find that!");
+      });
+  });
+});
```

Comment:
We should test for a category that exists but has no reviews yet. In this case we would expect a 200 and to return an empty array